### DISCUSSION TITLE
OSError: [Errno 24] Too many open files

### DISCUSSION THREAD

**Discussion by SWVG** [johanmeijer/grott#139] [Q&A]
I would like to run both grott server and growatt proxy on the same server (Raspberry 3B). For grott server, I changed the ip address in grott.ini to 0.0.0.0 and port 5279. Then I start the grott.server through python3 grott.server. That looks fine. However when (in a other session) I start grott through python3 grott.py. it shows many times - ('192.*.*.*’, 49130) has connected (where the 49130 number is then altered with 2 each time) and ends with the following error. 

Traceback (most recent call last):
  File "grott.py", line 35, in <module>
  File "/home/pi/grott/grottproxy.py", line 75, in main
  File "/home/pi/grott/grottproxy.py", line 90, in on_accept
  File "/home/pi/grott/grottproxy.py", line 31, in __init__
  File "/usr/lib/python3.7/socket.py", line 151, in __init__
OSError: [Errno 24] Too many open files

When I leave the grott server setting at default in grott,ini, all work fine.
Assuming I’m missing something, can anyone please advice what goes wrong? Same of course when there is already a solution or topic on this

**Comment by johanmeijer** [comment]
Grott is listening at port 5279. 
Grottserver listen at port 5781. You have to set grott server to 0.0.0.0 port 5781 in grott.ini I think..

**Comment by SWVG** [comment]
Hi Johan,

 

Dank je wel, het werkt nu wel 😊. Heb ik toch niet goed gelezen / gekeken. 

 

Groet, Steven

**Reply by SWVG** [reply]
Hi Johan, thank you very much. It's now working
